### PR TITLE
Create fr.lua

### PR DESCRIPTION
**Pull request by PoPoGH**
add french locale to esx_banking

### CODE REVIEW

**Review by CsokiHUN — DISMISSED** [review]

**Comment by PoPoGH** [comment]
> thank you for your pr. Maybe if you do the translation of [esx_banking/html/config.json](https://github.com/esx-framework/esx-legacy/blob/main/%5Besx_addons%5D/esx_banking/html/config.json), we will also be grateful for that

Done.

**Comment by CsokiHUN** [comment]
Good job, thanks!

**Review by CsokiHUN — APPROVED** [review]

File: [esx_addons]/esx_banking/html/config.json [hunk 1]
```diff
@@ -385,9 +385,106 @@
...
+                "elementID":"#wrapper",
+                "title":"",
+                "closeButtonText":"Fermer",
+                "bankTitle":"Banque Fleeca",
+                "mainExitButtonText": "Déconnexion"
+            },
+            {
+                "elementID":"#cpincode",
+                "name":"cpincode",
+                "buttonText":"Applique",
```
Comment (jbdevic) [inline [esx_addons]/esx_banking/html/config.json:406]: Valider

File: [esx_addons]/esx_banking/html/config.json [hunk 2]
```diff
@@ -385,9 +385,106 @@
...
+                "buttonText":"Approuver",
+                "inputPlaceholder": "Montant",
+                "title":"RETIRER",
+                "description": "Ici vous pouvez retirer votre argent de la banque.",
+                "type":"number"
+            },
+            {
+                "elementID":"#deposit",
+                "name":"deposit",
+                "buttonText":"Approbation",
```
Comment (jbdevic) [inline [esx_addons]/esx_banking/html/config.json:424]: Valider

File: [esx_addons]/esx_banking/html/config.json [hunk 3]
```diff
@@ -385,9 +385,106 @@
...
+                "buttonText":"Applique",
+                "inputPlaceholder": "Entrez 4 chiffres...",
+                "title":"CONFIGURATION DU PIN",
+                "description": "Ici vous pouvez définir ou changer votre code PIN.",
+                "type":"password"
+            },
+            {
+                "elementID":"#withdraw",
+                "name":"withdraw",
+                "buttonText":"Approuver",
```
Comment (jbdevic) [inline [esx_addons]/esx_banking/html/config.json:415]: Valider

File: [esx_addons]/esx_banking/html/config.json [hunk 4]
```diff
@@ -385,9 +385,106 @@
...
+                "buttonText":"Approbation",
+                "inputPlaceholder": "Montant",
+                "title":"DEPOSIT",
+                "description": "Ici, vous pouvez déposer votre argent à la banque.",
+                "type":"number"
+            },
+            {
+                "elementID":"#transfer",
+                "name":"transfer",
+                "buttonText":"Transfert",
```
Comment (jbdevic) [inline [esx_addons]/esx_banking/html/config.json:433]: Transférer

File: [esx_addons]/esx_banking/html/config.json [hunk 5]
```diff
@@ -385,9 +385,106 @@
...
+            "your_money_cash_label":"Votre argent en espèces",
+            "your_money_bank_label":"Votre solde bancaire",
+            "withdraw":"RETIRER",
+            "deposit":"DÉPOSER",
+            "transfer":"TRANSFERT",
+            "transferReceive":"TRANSFERT REÇU",
+            "moneyFormat":"$__replaceData__",
+            "graphTitle":"Votre solde",
+            "moreGraphTitle": "Grand graphique",
+            "moreHistoryTitle":"Afficher plus d'historique",
```
Comment (jbdevic) [inline [esx_addons]/esx_banking/html/config.json:474]: Afficher plus

File: [esx_addons]/esx_banking/locales/fr.lua [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Locales['fr'] = {
+    ['invalid_amount'] = "La somme d'argent est invalide.",
```
Comment (jbdevic) [inline [esx_addons]/esx_banking/locales/fr.lua:2]: Le montant est invalide.

**Review by jbdevic — APPROVED** [review]
some improvements

**Comment by PoPoGH** [comment]
Thx @jbdevic

**Pull request merged**